ReactWindows/ReactNative/Views/Text/ReactTextViewManager.cs

```diff
@@ -39,9 +40,39 @@ public override ReactTextShadowNode CreateShadowNodeInstance()
         /// <param name="extraData">The aggregated virtual node changes.</param>
         public override void UpdateExtraData(TextBlock root, object extraData)
         {
-            var inline = (Inline)extraData;
+            var textUpdate = (Tuple<Inline, TextAlignment?, double?, int?, int?>)extraData;
+
+            var inline = textUpdate.Item1;
+            var textAlignment = textUpdate.Item2;
+            var lineHeight = textUpdate.Item3;
+            var numberOfLines = textUpdate.Item4;
+            var letterSpacing = textUpdate.Item5;
+
             root.Inlines.Clear();
             root.Inlines.Add(inline);
+
+            if (letterSpacing.HasValue)
```

Comment:
One more thing, since we're reusing the TextBlock over and over again, if the nullable values are null, shouldn't we reset to the default values?  Or maybe the shadow node should just push non-nullable values since the settings will always need to be done anyway.